src/Symfony/Bridge/Doctrine/LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2004-2022 Fabien Potencier
+Copyright (c) 2004-2023 Fabien Potencier
```

Comment:
```
Copyright (c) 2004-present Fabien Potencier
```

What about using the "present" variant? This would indicate that the file was created or modified between 2004 and the present time. You would not need to update the copyright notice each year, as it already covers the current year and any future years.

It's also worth noting that the specific format of the copyright notice is not as important as the information it conveys. As long as you include the required information, you have some flexibility in how you present it.